content/en/user-guide/aws/stepfunctions/index.md

```diff
@@ -0,0 +1,127 @@
+
+LocalStack provides Step Functions support via the Community offering. You can use the Step Functions API to create, execute, update, and delete state machines. The supported APIs are available over our [feature coverage page]({{< ref "feature-coverage" >}}).
+
+## Getting started
+
+In this getting started guide, you'll learn how to make a basic usage of Step Functions over LocalStack. This guide is intended for users who wish to get more acquainted with Step Functions, and assumes you have basic knowledge of the AWS CLI (and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script). First, start your LocalStack instance using your preferred method, then run the following commands:
+
+1. Create a State Machine using the `create-state-machine` command and specify the name of the state machine, the Amazon States Language definition, and the role ARN to use for the state machine:
+   {{< command >}}
+   $ awslocal stepfunctions create-state-machine \
+        --name "WaitExecution" \
+        --definition '{
+        "StartAt": "WaitExecution",
+        "States": {
+            "WaitExecution": {
+            "Type": "Wait",
+            "Seconds": 10,
+            "End": true
+            }
+        }
+        }' \
+        --role-arn "arn:aws:iam::000000000000:role/stepfunctions-role"
+   {{< /command >}}
+   In the above example, we have created a state machine named `WaitExecution`, with a single state, `WaitExecution`, which waits for 10 seconds before completing the execution. The role ARN is used to grant the state machine access to other AWS services, and you can create an IAM role within LocalStack as well. The output of the above command is the ARN of the state machine:
+   ```json
+   {
+    "stateMachineArn": "arn:aws:states:<AWS_REGION>:000000000000:stateMachine:WaitExecution",
+    "creationDate": "<DATE>"
+   }
+   ```
+
+2. You can list the state machines in your account by running the following command:
+   {{< command >}}
+   $ awslocal stepfunctions list-state-machines
+   {{< /command >}}
+   The output of the above command is a list of state machines:
+   ```json
+   {
+    "stateMachines": [
+        {
+        "stateMachineArn": "arn:aws:states:<AWS_REGION>:000000000000:stateMachine:WaitExecution",
+        "name": "WaitExecution",
+        "type": "STANDARD",
+        "creationDate": "<DATE>"
+        }
+    ]
+   }
+   ```
+
+3. You can retrieve the State Machine ARN and query them using the State Machine name via the `list-state-machines` command:
+   {{< command >}}
+   $ awslocal stepfunctions list-state-machines \
+        --query 'stateMachines[?name==`WaitExecution`].stateMachineArn'
+   {{< /command >}}
+   The output of the above command is the state machine definition:
+   ```json
+   [
+        "arn:aws:states:<REGION>:000000000000:stateMachine:WaitExecution"
+   ]
+   ```
```

Comment:
Any specific reason for using `list-state-machines` again here instead of `awslocal stepfunctions describe-state-machine --state-machine-arn "..."` ?

With the current sample in step 3 this description also doesn't apply: `The output of the above command is the state machine definition:`

I'd replace this with something like this:

```suggestion
3. You can retrieve the state machine using the ARN returned in the first step, via the `list-state-machines` command:
   {{< command >}}
   $ awslocal stepfunctions describe-state-machine --state-machine-arn "arn:aws:states:<AWS_REGION>:000000000000:stateMachine:WaitExecution"
   {{< /command >}}
   The output of the above command is the state machine definition:
   ```json
   { TODO  }
   ```
```